src/Microsoft.ML.StandardLearners/Standard/SdcaStatic.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.ML.Data.StaticPipe.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Calibration;
+
+namespace Microsoft.ML.Runtime.Learners
+{
+    /// <summary>
+    /// Extension methods and utilities.
+    /// </summary>
+    public static class SdcaStatic
+    {
+        public static Scalar<float> PredictSdcaRegression(this Scalar<float> label, Vector<float> features, Scalar<float> weights = null,
```

Comment:
>PredictSdcaRegression [](start = 36, length = 21)

Let's get into habit of writing extensive summary comments for these, since this is our public API now

Reply:
I do

---
In reply to: [215430305](https://github.com/dotnet/machinelearning/pull/837#discussion_r215430305) [](ancestors = 215430305,215367489)